app/views/search/_sitetour_modal.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade sitetour" id="sitetourModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h1 class="modal-title" id="sitetourModalLabel">Welcome to Earthdata Search Client</h1>
+      </div>
+      <div class="modal-body" style="margin-left: 20px;">
+        <%= image_tag "nasa.png", :class => "pull-left"%>  <p>This application allows you to search, discover, visualize, refine, and access NASA Earth Observation data.  If you are new to this application, please follow the brief tour to get an overview of the features that will 
+        help you achieve your goals.</p>
+      </div>
+      <div class="modal-footer">
+        <a class="button text-button" id="closeInitialTourModal" data-dismiss="modal" href="#">Close</a>
+        <button class='button' data-bind="click: function(){ $root.ui.sitetour.startTour(); }">Start Tour <i class='fa fa-arrow-circle-right'></i></button>
+        <label class='pull-left' style='padding-top: 5px;padding-left: 5px;'>
+          <input class='toggleHideTour' id='toggleHideTour' type='checkbox' data-bind="click: function(){ return true; }, checked: $root.ui.sitetour.toggleHideTour(); "/>
```

Comment:
@mightynimble - I've the same question, now.  I feel like there was a purpose to that but the code doesn't seem to need it now.  Removing - thanks!